nixos/modules/services/video/frigate.nix

```diff
@@ -412,6 +412,7 @@ in
         intel-gpu-tools
       ];
       serviceConfig = {
+        ExecStartPre = "-rm /var/cache/frigate/*.mp4";
```

Comment:
```suggestion
        ExecStartPre = "rm /var/cache/frigate/*.mp4";
```

Reply:
This was intentional.

If you use `ExecStartPre=-` instead of `ExecStartPre=`, the pre command is not required for the actual service. And because e.g. on the first start of frigate the folder is empty, or if we restart it without any cameras in record mode, we need to address the fact that the folder could be empty and there are no files to delete.

With your change, Frigate wouldn't start in many setups.

Reply:
Ah, I see.